CordovaLib/include/Cordova/CDVViewController.h

```diff
@@ -32,11 +32,39 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ A view controller that specializes in displaying Cordova web content.
+
+ ## Overview
+ `CDVViewController` presents a view that displays Cordova web content in a
+ ``webView``. Although often presented as the root view controller for an app,
+ a `CDVViewController` can safely be placed within view controller
+ hierarchies—such as navigation and tabbed controllers—or presented modally.
+
+ The behavior preferences and web content to be loaded are defined in a Cordova
+ XML configuration file, for which a separate [reference
+ guide](https://cordova.apache.org/docs/en/dev/config_ref/index.html) exists.
```

Comment:
Is the `dev` version correct or should it be the `latest` release version?

Reply:
`latest` is probably better, I'll fix that